### DISCUSSION TITLE
default schema for __diesel_schema_migrations

### DISCUSSION THREAD

**Discussion by malvinpratama** [diesel-rs/diesel#4412] [Q&A]
can we change default schema for __diesel_schema_migrations?
because my company use microservice but 1 database. they service is separated by schema. so the __diesel_schema_migrations should be 1 per schema/service not just 1 in public.

**Comment by weiznich** [comment]
This is currently expected behavior and there is currently no way to change this. We might consider to accept a PR adding support for changing the schema of that table if someone comes up with a reasonable API for that.
